sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -220,6 +222,21 @@ public static Import importResources(
     return new Import(fhirStore, tempDir, deadLetterDir, contentStructure);
   }
 
+  /**
+   * Export resources to GCS. Intended for use on non-empty FHIR stores
+   *
+   * @param fhirStore the fhir store, in the format:
+   *     projects/project_id/locations/location_id/datasets/dataset_id/fhirStores/fhir_store_id
+   * @param exportGcsUriPrefix the destination GCS dir, in the format:
+   *     gs://YOUR_BUCKET_NAME/path/to/a/dir
+   * @return the export
+   * @see ExportGcs
+   */
+  public static ExportGcs exportResourcesToGcs(String fhirStore, String exportGcsUriPrefix) {
+    return new ExportGcs(
+        StaticValueProvider.of(fhirStore), StaticValueProvider.of(exportGcsUriPrefix));
+  }
```

Comment:
If you support valueproviders internally, you can add another method `exportResourcesToGcs` that receives `ValueProvider<String>` types - this will enable users to use them with themplates

Reply:
Nice. Thanks for catching. We indeed want to use this as a template :-)

Reply:
Done.